ansible-deployment/network-interfaces/tasks/main.yaml

```diff
@@ -61,3 +61,6 @@
 
      - name: Bring up inspector interface
        command: ifup inspector
+
+     - name: Add route for MOC IPMI network
+       copy: src=route-enp65s0f1np1 dest=/etc/sysconfig/network-scripts mode=0644
```

Comment:
Where is the 'ip route' command to add route-enp65s0f1np1 to controllers?

Reply:
adding the route using the `ip route` command is not persistent, which is why I switched to doing it via the network-scripts files so it would survive reboots.

Reply:
Gotcha. Thanks!